lib/jpegli/quant.cc

```diff
@@ -459,6 +459,26 @@ float DistanceToLinearQuality(float distance) {
   }
 }
 
+float DistanceToScale(float distance, int k) {
+  constexpr float kExponent[DCTSIZE2] = {
+      1.00f, 0.51f, 0.67f, 0.74f, 1.00f, 1.00f, 1.00f, 1.00f,  //
+      0.51f, 0.66f, 0.69f, 0.87f, 1.00f, 1.00f, 1.00f, 1.00f,  //
+      0.67f, 0.69f, 0.84f, 0.83f, 0.96f, 1.00f, 1.00f, 1.00f,  //
+      0.74f, 0.87f, 0.83f, 1.00f, 1.00f, 0.91f, 0.91f, 1.00f,  //
+      1.00f, 1.00f, 0.96f, 1.00f, 1.00f, 1.00f, 1.00f, 1.00f,  //
+      1.00f, 1.00f, 1.00f, 0.91f, 1.00f, 1.00f, 1.00f, 1.00f,  //
```

Comment:
perhaps a comment about the 0.91s

Reply:
I was a manually simplified simplex search result. I expect that it will change somewhat after I add frequency dependent dead-zone parameters.